[2022-04-13 22:34:50,231] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-03-02T06:00:13+00:00 [queued]>
[2022-04-13 22:34:50,252] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-03-02T06:00:13+00:00 [queued]>
[2022-04-13 22:34:50,253] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:34:50,254] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-04-13 22:34:50,254] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:34:50,326] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-03-02 06:00:13+00:00
[2022-04-13 22:34:50,366] {standard_task_runner.py:52} INFO - Started process 10673 to run task
[2022-04-13 22:34:50,389] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAG', 'ingest', 'scheduled__2022-03-02T06:00:13+00:00', '--job-id', '1805', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpiei16boz', '--error-file', '/tmp/tmpwvl7w_io']
[2022-04-13 22:34:50,406] {standard_task_runner.py:77} INFO - Job 1805: Subtask ingest
[2022-04-13 22:34:50,801] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-03-02T06:00:13+00:00 [running]> on host 13db72185839
[2022-04-13 22:34:51,084] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAG
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-03-02T06:00:13+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-02T06:00:13+00:00
[2022-04-13 22:34:51,088] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-13 22:34:51,093] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-04-13 22:34:51,139] {subprocess.py:85} INFO - Output:
[2022-04-13 22:34:51,178] {subprocess.py:89} INFO - ***.cfg
[2022-04-13 22:34:51,180] {subprocess.py:89} INFO - dags
[2022-04-13 22:34:51,181] {subprocess.py:89} INFO - logs
[2022-04-13 22:34:51,185] {subprocess.py:89} INFO - output.csv
[2022-04-13 22:34:51,187] {subprocess.py:89} INFO - plugins
[2022-04-13 22:34:51,191] {subprocess.py:89} INFO - requirements.txt
[2022-04-13 22:34:51,194] {subprocess.py:89} INFO - scripts
[2022-04-13 22:34:51,205] {subprocess.py:89} INFO - webserver_config.py
[2022-04-13 22:34:51,206] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-13 22:34:51,371] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAG, task_id=ingest, execution_date=20220302T060013, start_date=20220413T223450, end_date=20220413T223451
[2022-04-13 22:34:51,574] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-13 22:34:51,836] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
